packages/ember-glimmer/lib/utils/process-args.js

```diff
@@ -51,7 +51,9 @@ class SimpleArgs {
       let ref = namedArgs.get(name);
       let value = attrs[name];
 
-      if (ref[UPDATE]) {
+      if (typeof value === 'function') {
```

Comment:
In prior versions (1.13 - 2.8) we use a symbol stamped on the generated action function that we used as an identifier that this is an action.  

Assuming we do the same here (this would alleviate the concern about intentionally passing down a class / function that is intended to be two way bound), this would look something like:

```js
if (typeof value === 'function' && value[ACTION]) {
  attrs[name] = value;
} else if (ref[UPDATE]) {
  // ...snip...
}
```